templates/base.html

```diff
@@ -170,5 +171,24 @@
     <script src="{% static 'js/bootstrap.min.js' %}"></script>
     {% endcompress %}
     {% endblock %}
+    <!-- begin olark code -->
+    <script type="text/javascript" async>
```

Comment:
Javascript code should never be in the HTML template. Create a separate `.js` file an just include it.